indent issue with 1.1.0, worked with 0.24.1

I got following issue (in 1.1.0, worked perfectly in 0.24.1):

eslint config:

```
    "indent": [2, 2],
    "curly": [2, "multi-line"],
```

code #1:

```
4 module.exports =
5 {
6   'Unit tests':
7   {
```

output #1:

```
buster.js
   5:1  error  Expected indentation of 2 characters but found 0  indent
   6:3  error  Expected indentation of 4 characters but found 2  indent
   7:3  error  Expected indentation of 4 characters but found 2  indent
```

code #2:

```
11 var path     = require('path')
12  , crypto    = require('crypto')
13  ;
```

output #2:

```
   13:4  error  Expected indentation of 0 characters but found 2  indent
```

Thank you.

--- a/lib/rules/indent.js
+++ b/lib/rules/indent.js
@@ -350,18 +350,18 @@ module.exports = function(context) {
                 }
             }
             nodeIndent = getNodeIndent(effectiveParent);
-
             if (parentVarNode && parentVarNode.loc.start.line !== node.loc.start.line) {
                 if (parent.type !== "VariableDeclarator" || parentVarNode === parentVarNode.parent.declarations[0]) {
                     nodeIndent = nodeIndent + (indentSize * options.VariableDeclarator[parentVarNode.parent.kind]);
                 } else if (parent.loc.start.line !== node.loc.start.line && parentVarNode === parentVarNode.parent.declarations[0]) {
                     nodeIndent = nodeIndent + indentSize;
                 }
-            } else if (!parentVarNode && !isFirstArrayElementOnSameLine(parent) && effectiveParent.type !== "ExpressionStatement") {
+            } else if (!parentVarNode && !isFirstArrayElementOnSameLine(parent) && effectiveParent.type !== "ExpressionStatement" && effectiveParent.type !== "AssignmentExpression" && effectiveParent.type !== "Property") {
                 nodeIndent = nodeIndent + indentSize;
             }
 
             elementsIndent = nodeIndent + indentSize;
+
             checkFirstNodeLineIndent(node, nodeIndent);
         } else {
             nodeIndent = getNodeIndent(node);

--- a/tests/lib/rules/indent.js
+++ b/tests/lib/rules/indent.js
@@ -661,6 +661,27 @@ ruleTester.run("indent", rule, {
             "    };\n",
             options: [2, { VariableDeclarator: { var: 2, const: 3 }, "SwitchCase": 1}],
             ecmaFeatures: { blockBindings: true }
+        },
+        {
+            code:
+                "module.exports =\n" +
+                "{\n" +
+                "  'Unit tests':\n" +
+                "  {\n" +
+                "    rootPath: './',\n" +
+                "    environment: 'node',\n" +
+                "    tests:\n" +
+                "    [\n" +
+                "      'test/test-*.js'\n" +
+                "    ],\n" +
+                "    sources:\n" +
+                "    [\n" +
+                "      '*.js',\n" +
+                "      'test/**.js'\n" +
+                "    ]\n" +
+                "  }\n" +
+                "};",
+            options: [2]
         }
     ],
     invalid: [

